2022-03-13 00:16:31,046 INFO    MainThread:9106 [wandb_setup.py:_flush():71] setting env: {}
2022-03-13 00:16:31,046 INFO    MainThread:9106 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-13 00:16:31,047 INFO    MainThread:9106 [wandb_init.py:_log_setup():371] Logging user logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_001631-3qnfflwf/logs/debug.log
2022-03-13 00:16:31,047 INFO    MainThread:9106 [wandb_init.py:_log_setup():372] Logging internal logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_001631-3qnfflwf/logs/debug-internal.log
2022-03-13 00:16:31,047 INFO    MainThread:9106 [wandb_init.py:init():404] calling init triggers
2022-03-13 00:16:31,047 INFO    MainThread:9106 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-03-13 00:16:31,047 INFO    MainThread:9106 [wandb_init.py:init():460] starting backend
2022-03-13 00:16:31,047 INFO    MainThread:9106 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 00:16:31,053 INFO    MainThread:9106 [backend.py:ensure_launched():216] starting backend process...
2022-03-13 00:16:31,063 INFO    MainThread:9106 [backend.py:ensure_launched():221] started backend process with pid: 9769
2022-03-13 00:16:31,065 INFO    MainThread:9106 [wandb_init.py:init():469] backend started and connected
2022-03-13 00:16:31,071 INFO    MainThread:9106 [wandb_init.py:init():533] updated telemetry
2022-03-13 00:16:31,098 INFO    MainThread:9106 [wandb_init.py:init():563] communicating current version
2022-03-13 00:16:32,246 INFO    MainThread:9106 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 00:16:32,247 INFO    MainThread:9106 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-13 00:16:32,595 INFO    MainThread:9106 [wandb_init.py:init():606] starting run threads in backend
2022-03-13 00:16:37,605 INFO    MainThread:9106 [wandb_run.py:_console_start():1810] atexit reg
2022-03-13 00:16:37,606 INFO    MainThread:9106 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-13 00:16:37,606 INFO    MainThread:9106 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-13 00:16:37,610 INFO    MainThread:9106 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-13 00:16:37,610 INFO    MainThread:9106 [wandb_init.py:init():633] run started, returning control to user process
2022-03-13 00:16:54,032 INFO    MainThread:9106 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-13 00:16:54,036 INFO    MainThread:9106 [wandb_run.py:_restore():1752] restore
2022-03-13 00:16:56,636 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:16:56,859 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:16:57,175 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:16:57,617 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 806
}

2022-03-13 00:16:57,719 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15001
}

2022-03-13 00:16:57,822 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15001
}

2022-03-13 00:16:57,924 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15001
}

2022-03-13 00:16:58,027 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11015
  total_bytes: 15001
}

2022-03-13 00:16:58,129 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:58,232 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:58,334 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:58,436 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:58,538 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:59,065 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}

2022-03-13 00:16:59,898 INFO    MainThread:9106 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15001
  total_bytes: 15001
}
local_info {
}

2022-03-13 00:17:01,219 INFO    MainThread:9106 [wandb_run.py:_append_files():2180] logging synced files
